app.js

```diff
@@ -38,6 +38,43 @@ app.use(bodyParser.urlencoded({
 
 // Below are the middlewares for response body
 
+/**
+ * v1Error(err, req)
+ * Format the response for errors on v0.1 routes
+ *
+ * @param {error object} err
+ * @param {HTTP request} req
+ */
+function v1Error(err, req) { // eslint-disable-line no-unused-vars
+  return {
+    data: {
```

Comment:
So this one is still using the data wrap?

Reply:
Correct.